docs/man/containerd-config.toml.5.md

```diff
@@ -200,6 +211,48 @@ imports = ["/etc/containerd/runtime_*.toml", "./debug.toml"]
     rdt_config_file = ""
 ```
 
+### Multiple Runtimes
+
+The following is an example partial configuraton with two runtimes:
+
+```toml
+[plugins]
+
+  [plugins."io.containerd.grpc.v1.cri"]
+
+    [plugins."io.containerd.grpc.v1.cri".containerd]
+      default_runtime_name = "runc"
+
+      [plugins."io.containerd.grpc.v1.cri".containerd.runtimes]
+        [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
+          privileged_without_host_devices = false
+          runtime_type = "io.containerd.runc.v2"
+
+        [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options]
+           BinaryName = "/usr/bin/runc"
+
+        [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.other]
+          privileged_without_host_devices = false
+          runtime_type = "io.containerd.runc.v2"
+
+          [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.other.options]
+            BinaryName = "/usr/bin/path-to-runtime"
+```
+
+The above creates two runtimes - `runc` and `other` - and sets the default runtime to `runc`.
+The above are used _solely_ for runtimes invoked via CRI.
```

Comment:
```suggestion
The above are used _solely_ for runtimes invoked via CRI. To use the non-default "other" runtime in this example, a pod spec will include the runtime handler named "other" to specify the desire to use the named runtime config.
```

Reply:
I added it, but I don't get it. I had thought the options for passing a runtime were either the path or the URI-like reference (which gets converted to a binary name). 

This would answer my question from the README. Can you link to an example of using it this way, i.e. a named runtime?

Reply:
https://kubernetes.io/docs/concepts/containers/runtime-class/